### PR TITLE
Rework change tag buttons - part 1

### LINKED ISSUE #475: Optimize tag ui in Change Tags menu (Rename, Delete)
Rework the UI of each tag presented in the Change Tags menu. From a technical standpoint this could be a new web component.

Acceptance criteria:
* Use trash bin icon to delete tags
* Use edit icon to rename tags (replace previous mouseover functionality)

## Original proposal from @zhuiks 

I think that the editing should be in place. Without popup dialog. It can be triggered either by single/double tap (I think it's pretty common) or by "pencil" icon or both.
Instead of the checkbox, we can use "tag" icon that would be highlighted if the tag is assigned to the currently selected verse.

Знімок екрану з 2021-10-04 15-31-35

Знімок екрану з 2021-10-04 15-39-45
I also think that "garbage can" icon is better for tag deletion than the "x" icon. The "x" icon can be confused with tag unassigment action.

_Originally posted by @zhuiks in https://github.com/ezra-bible-app/ezra-bible-app/discussions/447#discussioncomment-1427073_

### PR DESCRIPTION
**Pull request by tobias-klein**
This closes #475.

The changes are rather simple and the primary target is to have a more accessible rename function for touch screens.
This is just a first step. A second step (in-place editing of tag titles) can be implemented based on #530.

* A fixed edit tag button was added to replace the mouseover "rename tag" functionality.
* The close icon has been replaced with the trashbin icon to make it visually more clear.

This is how it looks:
image

Note that the *tag list template* has been changed.
So when you check it out on your end you also need to `npm run compile-pug`.

### CODE REVIEW

**Comment by zhuiks** [comment]
@tobias-klein can you move tag-edit and tag-delete buttons after tag name and numbers.
So the first user attention will be on actual tag name instead of trash icon.

**Comment by tobias-klein** [comment]
> @tobias-klein can you move tag-edit and tag-delete buttons after tag name and numbers.
> So the first user attention will be on actual tag name instead of trash icon.

Good point. Do you have a CSS tip how this could be done while keeping the icons in "columns"?

**Comment by tobias-klein** [comment]
@zhuiks 
I moved the buttons to the right side.

image

With this it's a bit hard to the relationship between the buttons and the tag on the left side, maybe we should introduce alternating background colors for the items in the tag list?

**Comment by tobias-klein** [comment]
This is the latest variant with alternating background colors.

image

image

**Review by zhuiks — CHANGES_REQUESTED** [review]
Buttons can be styled differently. Smaller border radius, and just border without background.
But I can do it in or after #526. I think styling should be similar to side-panel buttons. 

Is there enough spacing between edit and delete buttons on the tablet?

So my only suggestion for this PR is to use CSS var for odd rows background.

**Review by zhuiks — DISMISSED** [review]

**Comment by tobias-klein** [comment]
> Buttons can be styled differently. Smaller border radius, and just border without background. But I can do it in or after #526. I think styling should be similar to side-panel buttons.

Let's change that separately.

> 
> Is there enough spacing between edit and delete buttons on the tablet?

Yes, I think so.

> 
> So my only suggestion for this PR is to use CSS var for odd rows background.

Done!

File: css/toolbox.css [hunk 1]
```diff
@@ -225,10 +225,28 @@ tr.td-underline td {
...
-  height: 1em;
-  margin-bottom: 1em;
+  height: 2em;
+  padding: 0.1em;
+  padding-top: 0.5em;
+  padding-left: 0.5em;
+}
+
+.checkbox-tag:nth-child(odd), .checkbox-tag.odd {
+  background: lightgray;
```
[the remark this answers is not in the document]
Comment (tobias-klein) [reply css/toolbox.css]: Done

**Pull request merged**